contrib/packaging/rpm/skydive.spec

```diff
@@ -9,6 +9,13 @@
 %define gotest() go test -compiler gc -ldflags "${LDFLAGS:-}" %{?**};
 %endif
 
+# SELinux
+%define selinux_relabel_files() \
```

Comment:
It seems you call %relabel_files (instead of selinux_relabel_files) if the %postun section

Reply:
I will merge skydive-selinux to skydive pkg